Makefile

```diff
@@ -105,7 +105,7 @@ else
 	$(call MINIKUBE_GO_BUILD_CMD,$@,$*)
 endif
 
-minikube_iso: # old target kept for making tests happy
+minikube_iso: $(shell find deploy/iso/minikube-iso -type f) image-bundle
```

Comment:
The dependencies should really be on this rule, since `out/minikube.iso` calls this rule.